sdk/src/Services/S3/Custom/Model/Internal/MarshallTransformations/AbortMultipartUploadRequestMarshaller.cs

```diff
@@ -50,6 +50,20 @@ public IRequest Marshall(AbortMultipartUploadRequest abortMultipartUploadRequest
             
             return request;
         }
+
+	    private static AbortMultipartUploadRequestMarshaller _instance;
```

Comment:
Is there a reason to not instantiate the marshaller here? That would avoid having to make null checks and will be consistent with the generated single implementations. Applies to all such S3 marshaller code.